plutus-tx/src/PlutusTx/Lift/TestInstances.hs

```diff
@@ -0,0 +1,110 @@
+{-# LANGUAGE ConstraintKinds          #-}
+{-# LANGUAGE DataKinds                #-}
+{-# LANGUAGE FlexibleContexts         #-}
+{-# LANGUAGE FlexibleInstances        #-}
+{-# LANGUAGE MultiParamTypeClasses    #-}
+{-# LANGUAGE PolyKinds                #-}
+{-# LANGUAGE QuantifiedConstraints    #-}
+{-# LANGUAGE StandaloneKindSignatures #-}
+{-# LANGUAGE TypeFamilies             #-}
+{-# LANGUAGE TypeOperators            #-}
+{-# LANGUAGE UndecidableInstances     #-}
+{-# LANGUAGE UndecidableSuperClasses  #-}
+
+module PlutusTx.Lift.TestInstances () where
+
+import PlutusCore qualified as PLC
+import PlutusCore.Builtin qualified as PLC
+import PlutusCore.Crypto.BLS12_381.G1 as G1 (Element)
+import PlutusCore.Crypto.BLS12_381.G2 as G2 (Element)
+import PlutusCore.Crypto.BLS12_381.Pairing (MlResult)
+import PlutusCore.Data
+import PlutusTx.Builtins
+import PlutusTx.Builtins.Class (FromBuiltin)
+import PlutusTx.Builtins.Internal (BuiltinBool, BuiltinList, BuiltinPair, BuiltinUnit)
+import PlutusTx.Lift.Class
+
+import Data.ByteString qualified as BS
+import Data.Kind qualified as GHC
+import Data.Text qualified as T
+
+import Prelude (Bool)
+
+-- | A class for converting each type from the universe to its @Builtin*@ counterpart. E.g.
```

Comment:
If this is useful maybe we should just make `FromBuiltin` stronger so it can encompass this?